### DISCUSSION TITLE
Meta Quest Pro VR headset

### DISCUSSION THREAD

**Discussion by Rodrigo-dev-bit** [1101-GS4719-C/Python-Community-GS4719-C#770] [General]
Meta 新出來的VR headset 差不多要$1500，應該是買不了。。。https://www.cnbc.com/2022/10/11/mark-zuckerberg-debuts-meta-quest-pro-vr-headset-that-will-cost-1500.html

**Comment by Pietra1226** [comment]
好酷，好想買來試試看

**Comment by StevenWanglolz** [comment]
這價格真的買不了，我乖乖盯著螢幕就好了
